### ISSUE TITLE
Syntax highlighting not supported for multiline strings.

### ISSUE THREAD

**Issue by phoenix-meadowlark** [jakoberzar/vscode-llvm-tablegen#1]
Currently multiline strings are not being highlighted as such. Here's an example taken from [the toy tutorial](https://mlir.llvm.org/docs/Tutorials/Toy/Ch-4/#intraprocedural-shape-inference)
```
def CastOp : Toy_Op<"cast", [NoSideEffect, SameOperandsAndResultShape]> {
  let summary = "shape cast operation";
  let description = [{
    The "cast" operation converts a tensor from one type to an equivalent type
    without changing any data elements. The source and destination types
    must both be tensor types with the same element type. If both are ranked
    then the rank should be the same and static dimensions should match. The
    operation is invalid if converting to a mismatching constant dimension.
  }];

  let arguments = (ins F64Tensor:$input);
  let results = (outs F64Tensor:$output);

  // Set the folder bit so that we can fold redundant cast operations.
  let hasFolder = 1;
}
```
Expected:
<img width="863" alt="Screen Shot 2020-04-14 at 12 41 19 PM" src="https://user-images.githubusercontent.com/40036383/79266992-4301e280-7e4d-11ea-988d-b72d033f64f4.png">

Current:
<img width="748" alt="Screen Shot 2020-04-14 at 12 41 42 PM" src="https://user-images.githubusercontent.com/40036383/79267058-5a40d000-7e4d-11ea-9e5e-824919f7b5ed.png">

**Comment by phoenix-meadowlark** [comment]
This also shows that numeric literals aren't being highlighted yet either on the `let hasFolder = 1;` line.

**Comment by jakoberzar** [comment]
Thank you for the notice.
The current version of the TextMate grammar that is used for syntax highlighting was taken directly from the LLVM repository. I checked if it's updated there, but it doesn't seem so.
I'll look into it later and try to fix it.

**Comment by jakoberzar** [comment]
I have updated the extension and published it as 0.0.2 to the VS Marketplace.
I have added support for integers, code fragments (multi-line strings) and some newer keywords.
@phoenix-meadowlark Please take a look and report if it's okay now :slightly_smiling_face:

**Comment by phoenix-meadowlark** [comment]
Looks amazing! Thanks for releasing this extension – it's _really_ nice to have syntax highlighting for such a niche language :)
